pkg/asset/installconfig/openstack/openstack.go

```diff
@@ -124,30 +125,29 @@ func Platform() (*openstack.Platform, error) {
 		return nil, err
 	}
 
+	trunkSupport := "0"
+	var i int
```

Comment:
why defining this here instead of in the else?

Reply:
I see, you are reusing it later... dismiss the comment

Reply:
ack.